tests/settings.py

```diff
@@ -0,0 +1,38 @@
+import dj_database_url
```

Comment:
There now appears to be `testing_settings.py` and `settings.py` in the same directory, this should be somehow dealt with

Reply:
So there's two attempts to run all tests against everything:
* with default settings and non swapped models `settings.py` 
* and with overwritten settings only that required by swapper - `testing_settings.py`

Reply:
Two attempts, but three files?

https://github.com/Akay7/fcm-django/blob/swapableDeviceModel/tests/settings.py
https://github.com/Akay7/fcm-django/blob/swapableDeviceModel/tests/testing_settings.py
https://github.com/Akay7/fcm-django/blob/swapableDeviceModel/tests/swap_settings.py

Reply:
Uh oh. Yeah, I removed extra.